pages/enroll/lti.ejs

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%- include('../partials/head', {pageTitle: 'Courses'}); %>
+
+    <style>
+      .modal-course-title-style {
+        font-size: 125%;
+      }
+    </style>
```

Comment:
This looks like it's unused on this page.

```suggestion
```

Reply:
@trombonekenny saw this comment was marked resolved - is there a reason this needs to be here?

Reply:
This is probably my fault of fixing the changes in a later commit and not addressing the review comments in GitHub directly. But I thought I fixed the issues?

Reply:
To be clear, the GitHub suggestion here is to remove this unused code; same with the other comment (https://github.com/PrairieLearn/PrairieLearn/pull/5281#discussion_r789961991). Per the diff, the code is still here.

Reply:
I must have missed a push or something, because these have been removed from my local copy. Does that look right to you now?

Reply:
This is what I see on the GitHub diff:

<img width="652" alt="Screen Shot 2022-03-14 at 12 46 36" src="https://user-images.githubusercontent.com/1476544/158249338-095a4346-4d6d-423d-84ee-b14b08ab56ee.png">

I also see it locally:

```
$ git diff master -- pages/enroll/lti.ejs
diff --git a/pages/enroll/lti.ejs b/pages/enroll/lti.ejs
new file mode 100644
index 000000000..73c30abee
--- /dev/null
+++ b/pages/enroll/lti.ejs
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%- include('../partials/head', {pageTitle: 'Courses'}); %>
+
+    <style>
+      .modal-course-title-style {
+        font-size: 125%;
+      }
+    </style>
+  </head>
+  <body>
+    <script>
+      $(function () {
+          $('[data-toggle="popover"]').popover({sanitize: false})
+      });
+    </script>
+    <%- include('../partials/navbar', {navPage: 'enroll'}); %>
+    <div id="content" class="container">
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white">Logout and log back in to see more courses</div>
+        <div class="card-body">
+            <p>Your PrairieLearn login is currently tied to <strong><%= lti_info.plc_short_name %>
+            <%= lti_info.ci_long_name %></strong> and cannot be used to enroll in other courses.
+            <p>
+            To see more courses:
+            <ol>
+                <li>Log out by selecting your name in the top right menu and selecting "Log out".</li>
+                <li>Sign-in again with your normal login.</li>
+                <li>Return to this enroll page to see the list of courses.</li>
+            </ol>
+            <p>Note: When you revisit the main <%= lti_info.plc_short_name %> course site and come
+                back to PrairieLearn from it, it will take over your login again. You might consider
+                using different web browsers for that course from your other PrairieLearn courses.
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```

When you look at https://github.com/PrairieLearn/PrairieLearn/pull/5281/files, do you still see the unneeded `<script>`/`<style>` tags?

Reply:
Everything looks good after b835bd8!